source/manage/command-line-tools.rst

```diff
@@ -6,43 +6,30 @@ Command line tools
 
 In self-managed deployments, a ``mattermost`` command is available for configuring the system from the directory where the Mattermost server is installed. For an overview of the Mattermost command line interface (CLI), `read this article <https://medium.com/@santosjs/plugging-in-to-the-mattermost-cli-8cdcef2bd1f6>`__ from Santos.
 
-These ``mattermost`` commands include the following functionality:
-
-**General administration**
+.. important::
 
--  Create teams
--  Create users
--  Assign roles to users
--  Reset user passwords
--  Invite users to teams
+  As of Mattermost v6.0, this CLI has been replaced with the `mmctl command line tool </manage/mmctl-command-line-tool.html>`__. However, `mattermost import </manage/command-line-tools.html#mattermost-import>`__ commands, `mattermost export </manage/command-line-tools.html#mattermost-export>`__ commands, and related subcommands, remain available and fully supported from Mattermost v6.0.
```

Comment:
@sadohert @streamer45 - Please confirm whether Mattermost continues to support the following CLI parent/child commands: `mattermost import` & `mattermost export`. If these are supported, I'll ensure they're re-added. 

If they're no longer supported, please review the change scope of https://github.com/mattermost/mattermost/pull/17995 and let me know if other CLI commands need to remain documented.

Reply:
Please also let me know if there's a specific Mattermost release version we want to tie all CLI deprecations to specifically via docs, if not Mattermost v6.0.

Reply:
@isacikgoz - Can you help answer these outstanding questions?

Reply:
import and export should be performed by `mmctl`

Reply:
@cwarnermm 

They are still part of the server binary so unless we remove them there (breaking change) we should probably keep their documentation. 

As far as I know only the `import bulk` and `export bulk` commands have been ported to `mmctl` which is the recommended way of using that functionality so we should likely deprecate the `mattermost` binary equivalents.

/cc @nab-77

Reply:
One more question about the ``mattermost import`` CLI command - one of it's documented child commands via ``--help`` is ``--slack``. Is this intended and desired?

Reply:
That's another command that we should propbably deprecate/remove. As far as I understand it's been replaced by the dedicated [`mmetl`](https://github.com/mattermost/mmetl) tool which can be used in conjunction with `mmctl` to achieve the same functionality the legacy command provided.

Reply:
Thank you so much for unblocking the remaining work needed for this docs update, @streamer45!